build.gradle

```diff
@@ -358,6 +361,17 @@ subprojects { subproject ->
     endToEndTestImplementation.extendsFrom runtimeOnly
   }
 
+  dependencyManagement {
+    imports {
+      mavenBom "org.springframework.cloud:spring-cloud-dependencies:${springCloudVersion}"
+    }
+  }
+
+  repositories {
+    mavenLocal()
+    mavenCentral()
+  }
```

Comment:
Does this need to be included here as it's already defined in allprojects?  If it does need to be included, should it have the same settings as the one for allprojects (e.g. including jitpack.io - see lines 193 to 202).

Reply:
Yes, subprojects need this as well.